netty/src/main/java/io/grpc/netty/NettyServerStream.java

```diff
@@ -53,6 +58,13 @@
   private final NettyServerHandler handler;
   private final Http2Stream http2Stream;
   private final WriteQueue writeQueue;
+  private final Attributes attributes;
+
+  private static Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+      Attributes.Key.named("remote-addr");
```

Comment:
Wow, how did I not realize that `Attributes` was only using the key name in its map. I like the changes to `Attributes` that @lukaszx0 has now made in this branch :)